src/editor/ui/UI.js

```diff
@@ -952,11 +952,14 @@ export default class UI {
     static createFormForText (p) {
         var tf = newHTML('div', 'pagetext off', p);
         tf.setAttribute('id', 'textbox');
-        if (isAndroid) {
-            tf.onmousedown = function (e) {
-                e.preventDefault();
-            };
-        }
+        // If the textbox is click or touched, the input may lost focus,
+        // and it may cause the text form miss closed.
```

Comment:
```suggestion
        // If the textbox background is clicked or touched, the input loses focus,
        // which causes the text input to close unexpectedly 
```
Just a suggestion for the comment.